esp/src/src-react/components/ProtectedBy.tsx

```diff
@@ -22,10 +36,8 @@ export const ProtectedBy: React.FunctionComponent<ProtectedByProps> = ({
 
     const [file, , , refreshData] = useFile(cluster, logicalFile);
     const [data, setData] = React.useState<any[]>([]);
-    const {
-        selection, setSelection,
-        setTotal,
-        refreshTable } = useFluentStoreState({});
+    const [userCount, setUserCount] = React.useState(0);
```

Comment:
Actually, this userCount state variable is redundant as well. You can change the destructured assignment of useFluentStoreState(...) below to also include total, ie `const { ..., total, setTotal, ...} = useFluentStoreState({});`

and then wherever you were using "userCount" just use "total" instead.